src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -844,27 +863,33 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
             $dumper->setProxyDumper(new ProxyDumper());
         }
 
+        $asFiles = $container->hasParameter('container.as_files') ? $container->getParameter('container.as_files') : true;
```

Comment:
container.dump_as_files?
or
container.dumper.as_files?

Reply:
Actually, I wanted to stick the current format

https://github.com/symfony/symfony/blob/64eaf7e3a63e6586f18582ea4ab8e5f334560525/src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php#L298-L300

Should I change? If so, I prefer `container.dumper.as_files`

Reply:
Double checking, `container.dumper.*` is for configuring the dumper,
while this one configures how the kernel uses the dumper...
So I'm for `container.dump_as_files`, which is the most explicit.

Reply:
okay, I will change this